dll/win32/riched20/txthost.c

```diff
@@ -761,6 +964,47 @@ static const ITextHost2Vtbl textHostVtbl =
     ITextHostImpl_QueryInterface,
     ITextHostImpl_AddRef,
     ITextHostImpl_Release,
+#ifdef _MSC_VER
+    ITextHostImpl_TxGetDC,
```

Comment:
What about using `STDCALL(ITextHostImpl_TxGetDC),` etc. as done in the table above, instead of adding the extra ifdef MSC_VER there (for example) as you have done:
```diff
+#ifdef _MSC_VER
+DECLSPEC_HIDDEN HDC __stdcall ITextHostImpl_TxGetDC( ITextHost2 *iface )
+#else
 DEFINE_THISCALL_WRAPPER(ITextHostImpl_TxGetDC,4)
 DECLSPEC_HIDDEN HDC __thiscall ITextHostImpl_TxGetDC( ITextHost2 *iface )
+#endif
{
```

Reply:
@HBelusca, I did try your suggestion and it did not work for me. Compiling with GCC gave this:
```
../dll/win32/riched20/txthost.c:971:5: error: implicit declaration of function '
STDCALL'; did you mean 'SIF_ALL'? [-Werror=implicit-function-declaration]
     STDCALL(ITextHostImpl_TxGetDC),
 ```